packages/components/src/components/tabs-bar/tabs-bar.tsx

```diff
@@ -243,9 +240,15 @@ export class TabsBar {
       case 'End':
         upcomingFocusedTabIndex = this.tabElements.length - 1;
         break;
-
-      case 'Enter':
-        this.onTabClick(focusedTabIndex);
+      // Tabindex elements within the ShadowDOM are improperly sorted into document's tabbing sequence, therefor the 'Tab' case needs to be handled.
```

Comment:
I think this isn't right.  
Better would be to mention the slotted buttons have a a different tabbing sequence in chrome and it appears that on hitting tab the first slotted one with `tabindex=0` becomes focused instead of the one after.